addons/mail/static/src/core/common/store_service.js

```diff
@@ -69,6 +45,10 @@ export class Store extends BaseStore {
     /** @type {typeof import("@mail/core/common/thread_model").Thread} */
     Thread;
 
+    get registeredImStatusPartners() {
+        return this.imStatusTrackedPersonas.map((persona) => persona.id);
+    }
```

Comment:
only if persona type is partner?

Reply:
This is already enforced by how `imStatusTrackedPersonas` is computed